pkgs/applications/misc/double-commander/default.nix

```diff
@@ -0,0 +1,104 @@
+{
+  stdenv, lib
+  , fetchFromGitHub
+  , fpc, lazarus
+  , which, autoPatchelfHook
+  , libqt5pas, qt5, gtk2, dbus
+}:
```

Comment:
```suggestion
{ lib
, stdenv
, fetchFromGitHub
, lazarus
, fpc
, libX11
, glib
, dbus

# GTK2
, gtk2
, atk
, pango
, gdk-pixbuf
, cairo

# Qt5
, libqt5pas
, qt5

, widgetset ? "qt5"
}:
```

Reply:
applied in fe0fe8d6b0ebe88ed7bff6309c3b786592a9d2bd